src/core/ComponentCell/index.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import { FormControlLabel } from "@mui/material";
+import { Args, Story } from "@storybook/react";
+import * as React from "react";
+import {
+  PrimaryText,
+  SecondaryText,
+  StyledTableData,
+} from "../BasicCell/style";
+import Checkbox from "../Checkbox";
+import InputToggle from "../InputToggle";
+import Tag from "../Tag";
+import ComponentCellRaw from "./index";
+
+const ComponentCell = (props: Args): JSX.Element => {
+  return (
+    <table>
+      <tbody>
+        <tr
+          style={{
+            display: "block",
+            maxWidth: "180px",
+          }}
+        >
+          <ComponentCellRaw
+            contentPosition="center"
+            data-testid="ComponentCell"
+            {...props}
+          />
+        </tr>
+      </tbody>
+    </table>
+  );
+};
+
+export default {
+  argTypes: {
+    contentPosition: {
+      control: { type: "select" },
+      options: ["left", "center", "right"],
+    },
+  },
```

Comment:
You can add the name for storybook here as well.

```suggestion
  },
  component: ComponentCell,
  title: "ComponentCell",
```

Reply:
ah thanks for this catch